lib/dpram.js

```diff
@@ -4,8 +4,8 @@ const indent = require('./indent.js');
 const get = require('lodash.get');
 
 const busDef = {
-  WRCLK: 1,
-  RDCLK: 1,
+  WRCLK: -1,
+  RDCLK: -1,
```

Comment:
I don't think I understand the discussion in #35 and why this is the right fix. Could I check my understanding here? I believe that `dpram` stands for dual-ported RAM and `spram` stands for single-ported RAM. This DUH bus interface definition says that the clock signals are _outputs_ of the memories. This seems strange to me.

Could you let me know if I'm wrong about any of the things described above, or if the above is all correct, then why we want the clocks going out of the module?

Reply:
The bus definition is here: https://github.com/sifive/duh-bus/blob/master/specs/sifive.com/MEM/DPRAM/0.1.0/DPRAM_rtl.json5#L14

All clocks are input on Master and Slave sockets

Reply:
oh I see I got positive/negative master/slave mixed up and misunderstood the issue. so after a closer look I think the issue is that connections are made according to the blackbox port direction and not the direction of the bus definition? I can update this PR to fix that